### ISSUE TITLE
Is onboard Audio working?

### ISSUE THREAD

**Issue by Emprint** [roederja/asus-rog-strix-b460I-hackintosh#5]
Hi @roederja,

Thank you very much for uploading your EFI and being brave to try a new generation motherboard.
With your successful experience, I'm also planning to buy the same board to build a small ITX config, without discrete GPU for now.
Can you tell me if onboard audio ports are working fine? I see on the motherboard documentation that they are using supposedly a custom HD Audio (ROG SupremexFX 7.1) and AppleALC.kext is more for standard codecs.
I won't need the 7.1 features but at least standard DP audio and audio input/output for a headset would be nice.

On my current hackintosh from 2013 I have to use an USB audio dongle as available solutions (Voodoo, AppleALC, AppleHDA, HDMI patches, etc) gave me trouble after upgrading to High Sierra but this was in the past.

**Comment by roederja** [comment]
Line out works fine (for stereo speakers - 2.0). Haven't tried the other ports, but they do show up in the sound control panel: 
![Input](https://user-images.githubusercontent.com/11135403/99801809-87193c00-2b2e-11eb-8c46-4b7f53d98c62.png)
![Output](https://user-images.githubusercontent.com/11135403/99801821-884a6900-2b2e-11eb-8267-3704f63b9aff.png)

**Comment by Emprint** [comment]
Thank you for your answer.
I see the following connections on the rear shield :
- Line In
- Line Out
- Mic In
- Sub
- Rear

Maybe some would only show if something is connected to it. Anyway I would not be using most of those, one input and one output is enough for most situations.
And there should be also connections for front panel but I think with your case you don't have front audio, maybe that is why it does not show.

**Comment by roederja** [comment]
I think they are all there. Line out on the shield -> Internal speakers. Sub and Rear are the other two Line out. Same for inputs I assume. Not sure about the front audio. Could be that they are just a different way to connect to the same port?

**Comment by Emprint** [comment]
Yes it makes sense, probably the front panel are just direct connection to the same.
Thanks.

On my previous hackintosh at some point I used a SSDT made by somebody else and the ports had labels like "Pink rear port", "Green port", etc.

I'm still hesitating between this motherboard and AsRock B460M-ITX that is 30% cheaper. (110 vs 160€), I'm still leaning towards this one though because it has 2 x M2 ports and USB C 20.

Thanks again for your help and for sharing your build.

**Comment by roederja** [comment]
That AsRock board seems very low budget. Only 4 USB ports and a PS/2 port instead and a silver I/O shild. I would discard that on looks alone ;)

**Comment by Emprint** [comment]
Yes it’s true it is a bit rough...

**Comment by Emprint** [comment]
As a follow up, I did not add the audio device-id in config.plist because with it I had audio issues in Big Sur (nothing present in the input/output devices), so removed it and just AppleALC.kext is enough.

@roederja I think you can close this issue

**Comment by dmikhalsky** [comment]
Is there a way to make DP audio work?

**Comment by roederja** [comment]
DP = DisplayPort? I don't know - don't have a device to test this with

**Comment by dmikhalsky** [comment]
Yes, DisplayPort. I have a monitor with built-in speakers. 
btw, thanks for TREMENDOUS job. I am very new to Hackintosh but everything is working from first try.

**Comment by Emprint** [comment]
I think Display Port audio works fine, I believe my father's display is using display port and has integrated HP and they do work.
